pkg/systemd/host.js

```diff
@@ -567,6 +580,35 @@ PageServer.prototype = {
         self.client = null;
 
         $(cockpit).off('.server');
+
+        $(self.sysroot).off();
+        self.sysroot = null;
+        if (self.ostree_client) {
+            self.ostree_client.close();
+            self.ostree_client = null;
+        }
+    },
+
+    sysroot_changed: function() {
```

Comment:
Can this code be done the same way we do realmd? Have the code in the ostree module and conditionally bring it in based on the manifest?